h/static/scripts/directive/test/excerpt-test.js

```diff
@@ -2,10 +2,24 @@
 
 var angular = require('angular');
 
-var assign = require('core-js/modules/$.object-assign');
 var util = require('./util');
 var excerpt = require('../excerpt');
 
+/**
+ * Wait for an <excerpt> to recompute its overflowing state.
+ *
+ * This happens asynchronously after an <excerpt> is created in order to wait
+ * for Angular directives used by the <excerpt>'s content to fully resolve.
+ *
+ * @return {Promise}
+ */
+function waitForLayout(element) {
+  element.scope.$digest();
+  return new Promise(function (resolve) {
+    window.requestAnimationFrame(resolve);
```

Comment:
`window.requestAnimationFrame` is now available in all browsers that we support (IE >= 10, Safari >= 6, Chrome/Firefox for a very long time), so we don't need to polyfill it any more. Having said that, having an adapter makes for cleaner tests. I'll have a think on this.